docs/rules/classic-decorator-hooks.md

```diff
@@ -26,7 +26,7 @@ export default class MyService extends Service {
 @classic
 export default class MyService extends Service {
   constructor() {
-    super();
+    super(...arguments);
```

Comment:
This would be slightly better to be compatible with [prefer-rest-params](https://eslint.org/docs/rules/prefer-rest-params):

```
constructor(...args) {
  super(...args);
  // ...
}
```